sklearn/utils/testing.py

```diff
@@ -375,7 +375,7 @@ def assert_raise_message(exceptions, message, function, *args, **kwargs):
                              (names, function.__name__))
 
 
-def assert_allclose_dense_sparse(x, y, rtol=1e-07, atol=0, err_msg=''):
+def assert_allclose_dense_sparse(x, y, rtol=1e-07, atol=1e-9, err_msg=''):
```

Comment:
Actually should we keep the same default as np.assert_allclose i.e. atol=0 and just use atol != 0 in the tests that need it?